Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -138,7 +138,12 @@ function createModelFeatureTables(model, featureMetadata) {
   return modelFeatureTables;
 }
 
-function selectFeatureTableId(components, model) {
+function selectFeatureTableId(components, model, content) {
+  // For 3D Tiles 1.0 formats, the feature table always has the "_batchTable" feature table.
+  if (defined(content.featureMetadata)) {
+    return "_batchTable";
```

Comment:
It would be less error prone if `"_batchTable"` was a static property of some class